peregrine/initSettings.py

```diff
@@ -13,7 +13,7 @@ class initSettings:
   def __init__(self):
 
     self.msToProcess          = 39000
```

Comment:
Having some comments here over what these numbers mean would be nice. Wouldn't you want `msToProcess` to be an argument to `__init__`.